e2e/cypress/plugins/index.js

```diff
@@ -1,17 +1,10 @@
-// ***********************************************************
-// This example plugins/index.js can be used to load plugins
-//
-// You can change the location of this file or turn off loading
-// the plugins file with the 'pluginsFile' configuration option.
-//
-// You can read more here:
-// https://on.cypress.io/plugins-guide
-// ***********************************************************
+const cypressTypeScriptPreprocessor = require('./cy-ts-preprocessor')
 
 // This function is called when a project is opened or re-opened (e.g. due to
 // the project's config changing)
 
-module.exports = (on, config) => {
+module.exports = on => {
```

Comment:
[nit] Either we keep the `config` bit or we remove the comment below explaining it, the proposed halfway thing is confusing :wink:

Reply:
hmm yeah the plugin changed that for reasons unknown. not sure if we'll ever need `config` but i'll drop the comments for now